docs/source/upgrade.md

```diff
@@ -2,6 +2,48 @@
 
 This pages describes Scylla Operator upgrade procedures.
 
+## `v1.0.0` -> `v1.1.0`
+
+During this update we will change readiness probes for Scylla Operator.
+The new version brings an automation for upgrade of sidecar image, so a rolling restart of managed Scylla clusters is expected.
+
+1. Get name of StatefulSet managing Scylla Operator
+   ```shell
+   kubectl --namespace scylla-operator-system get sts --selector="control-plane=controller-manager"
+
+   NAME                                 READY   AGE
+   scylla-operator-controller-manager   1/1     95m
+   ```
+
+1. Change probes and used container image by applying following patch:
+    ```yaml
+    apiVersion: apps/v1
+    kind: StatefulSet
```

Comment:
patch isn't GVK aware, I'd assume you only need the spec part

Reply:
right, removed.